src/PanoViewer/PanoViewer.js

```diff
@@ -170,6 +170,10 @@ export default class PanoViewer extends Component {
 			return this;
 		}
 
+		// Check XR support at not when imported, but when created.
+		// This is for easier use of polyfills
+		checkXRSupport();
```

Comment:
@WoodNeck 

Is there an error if called more than once?

If so, you need to retest. If not, there is no need to recheck.

Reply:
I don't think that there can be an error when it called multiple times.
But, you're right that there's no need to check it again.